dev_setup.sh

```diff
@@ -338,17 +338,17 @@ else
     # first, look for a build of mimic in the folder
     has_mimic=""
     if [[ -f ${TOP}/mimic/bin/mimic ]] ; then
-        has_mimic=$( ${TOP}/mimic/bin/mimic -lv | grep Voice ) || true
+        has_mimic=$(${TOP}/mimic/bin/mimic -lv | grep Voice) || true
     fi
 
     # in not, check the system path
-    if [ "$has_mimic" == "" ] ; then
-        if [ -x "$(command -v mimic)" ] ; then
-            has_mimic="$( mimic -lv | grep Voice )" || true
+    if [[ -z $has_mimic ]] ; then
+        if [[ -x "$(command -v mimic)" ]] ; then
```

Comment:
Can these quotation marks be removed as well?

Reply:
@forslund 
Ah jep makes sense, good to be consistent then.